README.md

```diff
@@ -12,13 +12,15 @@ This GitHub action will build your [Hugo site](https://gohugo.io/), and then pub
 
 - `GITHUB_ACTOR`: The name of the person or app that initiated the workflow. For example, octocat. [See here](https://developer.github.com/actions/creating-github-actions/accessing-the-runtime-environment/#environment-variables).
 - `TARGET_REPO`: This is the repo slug for the GitHub pages site. e.g. `benmatselby/benmatselby.github.io`.
+- `TARGET_BRANCH`: This is the branch to push the public files e.g. `docs`. Default is `master` branch.
 - `HUGO_VERSION`: This allows you to control which version of Hugo you want to use. There is a default within the action, but this may be out of date.
 - `HUGO_EXTENDED`: If set to `true`, the _extended_ version of Hugo will be used. Default is `false`.
 - `HUGO_ARGS`: Arguments passed to `hugo`.
 - `CNAME`: Contents of a `CNAME` file.
 
 ## Example
 
+### Publish to master branch
```

Comment:
Can we remove this please, if we aren't adding another one?

Reply:
Removed